<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>566</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="generate_button">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(172, 172, 172);</string>
        </property>
        <property name="text">
         <string>Gerar QR Code</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scan_button">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(172, 172, 172);</string>
        </property>
        <property name="text">
         <string>Escanear QR Code</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
